.github/workflows/acceptance_test.yml

```diff
@@ -33,7 +33,10 @@ jobs:
       - name: "[Pull Request] Get commit message"
         if: github.event_name == 'pull_request'
         id: pr_get_commit_message
-        run: echo "pr_commit_message=\"$(git log --format=%B -n 1 HEAD^2)\"" >> $GITHUB_OUTPUT
+        # Obtain the last commit from the branch to merge (hence the HEAD^2).
+        # In case of multi-line commit messages, remove any \n, because the GITHUB_OUTPUT method
+        # of sending data to other jobs does not like them.
+        run: echo "pr_commit_message=\"$(git log --format=%B -n 1 HEAD^2 | tr '\n' ' '))\"" >> $GITHUB_OUTPUT
```

Comment:
Have you tried like this?
```
echo "pr_commit_message=$(git log --format=%B -n 1 HEAD | tr '\n' ' '))"
```
... because in terminal, it fails with the `^2`. And the extra "" are not necessary, I've confirmed this.

Reply:
The HEAD^2 works for a merge. It takes the previous message from the branch that will be merged.
HEAD for a merge would be the merge message itself (Merge branch ... into ...)

Reply:
Ah, I didn't know that. J'va me coucher moins niaiseux ;-)

Reply:
Removed the quotes.